src/python/pants/backend/experimental/adhoc/register.py

```diff
@@ -17,3 +19,11 @@ def rules():
         *adhoc_tool.rules(),
         *run_system_binary.rules(),
     ]
+
+
+def build_file_aliases():
+    return BuildFileAliases(
+        objects={
+            "_runnable": runnable,
```

Comment:
Why `_runnable`? What is the thought process behind this name?

Reply:
Moot.